### ISSUE TITLE
Trimble 3603

### ISSUE THREAD

**Issue by VERVALL** [robots/gdm#8]
Muchas gracias por tu trabajo ROBOTS.
Mi pregunta es, funcionaría tú código en una estación Trimble serie 3600?

**Comment by robots** [comment]
What exactly do you mean? and if possible write in english :-)

**Comment by VERVALL** [comment]
Excuse me, I mean, if your code can work on a trimble 3600 series, with an alphanumeric CU, I have to change the internal batteries and I would like to be able to make a copy of the software.
Thanks Oscar

**Comment by robots** [comment]
No, there is no way to backup the CU right now. Not even for 600 station
